src/test/scala/org/hammerlab/magic/rdd/CachedCountRegistryTest.scala

```diff
@@ -0,0 +1,42 @@
+package org.hammerlab.magic.rdd
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.CachedCountRegistry
+import org.apache.spark.CachedCountRegistry._
+
+import org.hammerlab.magic.util.SparkSuite
+
+class CachedCountRegistryTest extends SparkSuite {
+  test("single rdd count") {
+    CachedCountRegistry.resetCache()
```

Comment:
nit: move this to a `before()` method? cf. [`BeforeAndAfter.before`](https://github.com/scalatest/scalatest/blob/release-2.2.2-for-scala-2.11-and-2.10/src/main/scala/org/scalatest/BeforeAndAfter.scala#L135).